### ISSUE TITLE
Missing permissions when installing from the marketplace

### ISSUE THREAD

**Issue by kierenevans** [RocketChat/Apps.Dialogflow#63]
Rocket Chat version: v3.16.0

Upon installing from the marketplace, after configuring settings in the app's settings, the "onSettingsUpdated" event logs return:
```
debug: 2021-06-29T13:21:19.475Z Caller: anonymous OR constructor
[
  "onSettingUpdated is being called..."
]

error: 2021-06-29T13:21:19.481Z Caller: run -> run
[
  "Error occurred while sending a HTTP Request"
]

debug: 2021-06-29T13:21:19.481Z Caller: anonymous OR constructor
[
  "'onSettingUpdated' was successfully called! The result is:",
  null
]
```
Digging into the server logs I see:
```
server.js:204 API ➔ debug POST: /api/apps/21b7d3ba-031b-41d9-8ff2-fbbfa081ae90/settings
Error: Failed to call the method  as the app (21b7d3ba-031b-41d9-8ff2-fbbfa081ae90) lacks the following permissions:
["networking"]. Declare them in your app.json to fix the issue.
reason: undefined
    at AppHttpBridge.<anonymous> (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/bridges/HttpBridge.js:20:22)
    at Generator.next (<anonymous>)
    at /app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/bridges/HttpBridge.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/bridges/HttpBridge.js:4:12)
    at AppHttpBridge.doCall (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/bridges/HttpBridge.js:19:16)
    at Http.<anonymous> (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/accessors/Http.js:60:63)
    at Generator.next (<anonymous>)
    at /app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/accessors/Http.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/accessors/Http.js:4:12)
    at Http._processHandler (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/accessors/Http.js:37:16)
    at Http.post (/app/bundle/programs/server/npm/node_modules/@rocket.chat/apps-engine/server/accessors/Http.js:28:21)
    at DialogflowClass.generateNewAccessToken (evalmachine.<anonymous>:35:41)
    at OnSettingUpdatedHandler.run (evalmachine.<anonymous>:21:43)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
server.js:204 API ➔ debug GET: /api/apps/21b7d3ba-031b-41d9-8ff2-fbbfa081ae90?marketplace=true&update=true&appVersion=1.2.2
server.js:204 API ➔ debug GET: /api/apps/21b7d3ba-031b-41d9-8ff2-fbbfa081ae90/apis
server.js:204 API ➔ debug Success {
  statusCode: 200,
  body: { apis: [ [Object], [Object] ], success: true }
}
```

Compiling the application locally and installing the zip works, presumably due to the permissions not being checked for self-compiled apps?

**Comment by murtaza98** [comment]
I believe this was a common bug across all omnichannel apps for the `3.16.0` version which has been fixed in the `3.16.1` version. [Release notes](https://github.com/RocketChat/Rocket.Chat/releases/tag/3.16.1).

**Comment by kierenevans** [comment]
Yes things seem to be better now, thank you :)
